enclave_wrangler/objects.py

```diff
@@ -90,34 +76,54 @@ def objects(self, object_type) -> pd.DataFrame:
             results += response_json['data']
             if 'nextPageToken' not in response_json or not response_json['nextPageToken']:
                 break
-            url = url_base + '?pageToken=' + response_json["nextPageToken"]
+            url = first_page_url + '?pageToken=' + response_json["nextPageToken"]
+        return results, response
+
+
+    # todo?: Need to find the right object_type, then write a wrapper func around this to get concept sets
+    #  - To Try: CodeSystemConceptSetVersionExpressionItem, OMOPConcept, OMOPConceptSet, OMOPConceptSetContainer,
+    #    OmopConceptSetVersionItem
+    def get_objects_by_type(
```

Comment:
Renamed `EnclaveClient.objects()` to `EnclaveClient.get_objects_by_type()` to make it clearer what this does.

Reply:
Also updated parameterization. Was previously just `object_type`, but now has: ` object_type: str, save_csv=True, save_json=True, outdir: str = None`